staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5692,6 +5692,7 @@ type ServiceAccount struct {
 
 	// Secrets is a list of the secrets in the same namespace that pods running using this ServiceAccount are allowed to use.
 	// Pods are only limited to this list if this service account has a "kubernetes.io/enforce-mountable-secrets" annotation set to "true".
+	// The "kubernetes.io/enforce-mountable-secrets" annotation is deprecated since v1.32.
```

Comment:
```suggestion
	// The "kubernetes.io/enforce-mountable-secrets" annotation is deprecated since v1.32.
	// Use separate namespaces if isolation of mounted secrets is desired.
	// See the docs at <LINK> for more details.
```